themes/metabolic/css/_bs_variables_theme.scss

```diff
@@ -0,0 +1,220 @@
+	$font-size-sm:                $font-size-base * .875 !default;
+	$font-size-lg:                $font-size-base * 1.25 !default;
+	
+	// $font-family-sans-serif: "Lato", "Noto Sans" !default;
+$font-family-sans-serif: "Raleway", sans-serif;
+	
+	$font-weight-lighter:         lighter !default;
+	$font-weight-light:           300 !default;
+	$font-weight-normal:          300 !default;
+	$font-weight-medium:          400 !default;
+	$font-weight-semibold:        600 !default; /* lato doesn't have 600 */
+	$font-weight-bold:            700 !default;
+	$font-weight-bolder:          bolder !default;
+	
+	$font-weight-base:            $font-weight-normal !default;
+	
+	$h1-font-size:                $font-size-base * 2 !default;
+	$h2-font-size:                $font-size-base * 1.75 !default;
+	$h3-font-size:                $font-size-base * 1.5 !default;
+	$h4-font-size:                $font-size-base * 1.25 !default;
+	$h5-font-size:                $font-size-base * 1 !default;
+	$h6-font-size:                $font-size-base * .75 !default;
+	// scss-docs-end font-sizes
+	
+	// scss-docs-start headings-variables
+	$headings-margin-bottom:      $spacer * .5 !default;
+$headings-font-family:        "Crimson Text", sans-serif;
+	$headings-font-style:         null !default;
+	$headings-font-weight:        400 !default;
+	$headings-line-height:        1.2 !default;
+$headings-color:        $gray-600;
+	// scss-docs-end headings-variables
+	
+	// scss-docs-start display-headings
+	$display-font-sizes: (
+	  1: 4rem,
+	  2: 3.5rem,
+	  3: 3rem,
+	  4: 2.5rem,
+	  5: 2rem,
+	  6: 1.5rem
+	) !default;
+	
+$display-font-family: "Crimson Text", sans-serif;
+	$display-font-style:  null !default;
+	$display-font-weight: 300 !default;
+	$display-line-height: $headings-line-height !default;
+	// scss-docs-end display-headings
+	
+	// scss-docs-start font-sizes
+	$font-sizes: (
+	  1: $h1-font-size,
+	  2: $h2-font-size,
+	  3: $h3-font-size,
+	  4: $h4-font-size,
+	  5: $h5-font-size,
+	  6: $h6-font-size
+	) !default;
+	
+$body-color: $gray-600;
+	$body-secondary-color:      rgba($body-color, .75) !default;
+	$body-secondary-bg:         $gray-300 !default; /* used for .bg-body-secondary */
+	
+	$body-tertiary-color:       rgba($body-color, .5) !default;
+	$body-tertiary-bg:          $gray-200 !default; /* used for .bg-body-tertiary */
+	
+	$light-bg-subtle:         mix($gray-100, #FFF) !default;
+	$dark-bg-subtle:          $gray-400 !default;
+	
+	$box-shadow:                  0 1px 8px rgba(#000, .15) !default;
+	
+	// scss-docs-start border-radius-variables
+	$border-radius:	0 !default;
+	
+	/* links */
+$link-color: $gray-600;
+	$link-decoration: underline solid .5px  !default;
+	$link-shade-percentage: 20% !default;
+	/*$link-hover-color: #000 !default;*/
+	$link-hover-decoration: null !default;
+	
+	
+	/* nav bar */
+	
+$nav-link-padding-y:                33px;
+$nav-link-padding-x:                24px;
```

Comment:
You don't need to change this, but try to use em or rem instead of px when possible otherwise you have to adjust if the overall font size changes.